docs/fast-pipe.md

```diff
@@ -103,3 +103,28 @@ We turn this into:
 ```reason
 let result = Some(preprocess(name))
 ```
+
+## Pipe Placeholders
+
+Sometimes you don't want to pipe the value you have into the first position. In these cases you can mark a placeholder value to show which argument you would like to pipe into.
+
+Let's say you have a function `namePerson`, which takes a `person` then a `name` argument. If you are transforming a person then pipe will work as-is:
+
+```reason
+makePerson(~age=47, ())
+  ->namePerson("Jane");
+```
+
+But if you have a name that you want to apply to a person object, you can use a placeholder:
```

Comment:
You should show what a placeholder is first and then show how to use it later, then it will become more clear. It will also make it more clear if you don't start the sentence with "But" here in this case when explaining a new concept.